recipe/meta.yaml

```diff
@@ -57,7 +59,7 @@ test:
     - psutil
 
 about:
-  home: http://www.pycryptodome.org
+  home: https://www.pycryptodome.org
   license: Public Domain/BSD 2-Clause/OCB/MPIR
```

Comment:
The licenses should be: `Unlicense AND BSD-2-Clause`, see https://github.com/Legrandin/pycryptodome/blob/v3.15.0/LICENSE.rst

Reply:
well, actually Unlicense is PD ... I consider them as equivalent